admin-ui/plugins/services/Components/Configuration/CacheInMemory.test.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
```

Comment:
The single quote is used everywhere not a double quote.

Reply:
Fixed